doc/api/net.md

```diff
@@ -691,6 +691,47 @@ added: v0.1.90
 Emitted when a socket connection is successfully established.
 See [`net.createConnection()`][].
 
+### Event: `'connectionAttempt'`
+
+<!-- YAML
+added: REPLACEME
+-->
+
+* `ip` {number} The IP which the socket is attempting to connect to.
```

Comment:
@ShogunPanda Looks like it should be `* address {string}`. Not `number`.

Reply:
You are right. This has been fixed in #51490.